site/pages/what-we-do/lean-agile-slice/style.css

```diff
@@ -0,0 +1,355 @@
+@value badgerRed, badgerBlack, mauve, badgerWhite from "../../../css/_colors.css";
+@value largeScreen, mediumScreen from "../../../css/_sizes.css";
+
+.leanAgileSlice {
+  padding: 40px 0;
+  background: white;
+}
+
+.leanAgileHeader {
+  composes: fontM from "../../../css/typography/_fonts.css";
+  composes: serif from "../../../css/typography/_fonts.css";
+  max-width: 210px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.loopContainer {
+  position: relative;
+  height: 124px;
+  margin: 30px auto;
+  text-align: center;
+}
+
+.word {
+  composes: fontS from "../../../css/typography/_fonts.css";
+  composes: serif from "../../../css/typography/_fonts.css";
+  width: 130px;
+  position: absolute;
+  top: 50%;
+  -webkit-transform: translate(0%, -50%);
```

Comment:
we have autoprefixer https://github.com/redbadger/website-honestly/blob/master/package.json#L11 so prefixes are not needed :)